ci/ciimage/fedora/install.sh

```diff
@@ -6,7 +6,7 @@ source /ci/common.sh
 
 pkgs=(
   python python-pip python3-devel
-  ninja-build make git autoconf automake patch
+  ninja-build make git autoconf automake patch file
```

Comment:
Thanks, I had noticed a similar thing recently but not gotten around to fixing this.

Apparently Fedora decided to remove it from their base images, which is a fascinating thought...